src/styles/variables.js

```diff
@@ -133,5 +133,5 @@ export default {
     // This is calculated based on the values specified in the 'getGoogleListViewStyle' function of the 'StyleUtils' utility
     googleEmptyListViewHeight: 14,
     hoverDimValue: 0.5,
-    pressDimValue: 0.2,
+    pressDimValue: 0.8,
```

Comment:
pardon my ignorance, this is needed because?

Reply:
that's what the design team wants, feel free to checkout the PR discussion & comments to get more context.

Reply:
Got it sorry about that, I was still going through the comments. The tests, or proposal doesn't cover this change. Hence my confusion.